crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1156,6 +1156,19 @@ impl World {
         result
     }
 
+    /// "Sends" an `event`.
```

Comment:
```suggestion
    /// "Sends" an [event](crate::event).
```

Linking to module-level documentation, where in the future it will probably contain some general info about how the event API works.

Reply:
Module level docs are better than my suggestion of the event type.